cabby/geo/map_processing/map_structure.py

```diff
@@ -53,29 +50,33 @@ def __init__(self, map_name: Text, level: int, load_directory: Text = None):
 
         elif map_name == "Pittsburgh":
             self.polygon_area = box(
-                miny=40.425, minx=-80.035, maxy=40.460, maxx=-79.930, 
+                miny=40.425, minx=-80.035, maxy=40.460, maxx=-79.930,
                 ccw=True)
-
         else:  # Bologna.
             self.polygon_area = box(
-                miny=44.4902, minx=11.3333, maxy=44.5000, maxx=11.3564, 
+                miny=44.4902, minx=11.3333, maxy=44.5000, maxx=11.3564,
                 ccw=True)
 
-
         if load_directory is None:
             self.poi, self.streets = self.get_poi()
-            self.nx_graph = ox.graph_from_polygon(self.polygon_area, network_type = 'all')
-            self.nodes, _ = ox.graph_to_gdfs(self.nx_graph)
-            
-            # Find closest nodes to POI.
-            self.poi['node'] = self.poi['centroid'].apply(lambda x: 
-                ox.distance.get_nearest_node(self.nx_graph, 
-                    util.tuple_from_point(x)))
+            self.nx_graph = ox.graph_from_polygon(
+                self.polygon_area, network_type='walk')
+            self.nodes, self.edges = ox.graph_to_gdfs(self.nx_graph)
 
+            # Find closest nodes to POI.
+            self.poi['node'] = self.poi['centroid'].apply(lambda x:
+                                                          ox.distance.
+                                                          get_nearest_node(self.
+                                                          nx_graph, util.
+                                                          tuple_from_point(x)))
+                                                                                
```

Comment:
Rather than breaking this up over multiple lines, it would be better to define a function and apply it here.